### PR TITLE
Cleanup commented code from muon template

### PR DESCRIPTION
**Pull request by maxnoe**

### CODE REVIEW

File: config_LST1_muons.cfg [hunk 1]
```diff
@@ -100,53 +79,38 @@ ARRANG -4.84             // La Palma, 01/12/2021
...
 *
 * LSTs:
-*
-* #define LSTHT  16.00E2
-* #define LSTRAD 12.50E2
-* 
-* TELESCOPE    0.0E2    0.0E2  $(LSTHT)    $(LSTRAD)  # LST
-TELESCOPE    0.0E2    0.0E2  16.00E2    12.50E2  // LST
+TELESCOPE 0.0E2    0.0E2  16.00E2    12.50E2  // LST
 CSCAT  1  9.8E2  0.     # To 80% of dish radius
```
Comment (maxnoe) [inline config_LST1_muons.cfg]: Should we make the scatter radius here configurable? There is surely a template option for this, right?

Comment (Voutsi) [reply config_LST1_muons.cfg]: Indeed there is. We can make NSCAT configurable as well

File: config_LST1_muons.cfg [hunk 2]
```diff
@@ -100,53 +80,37 @@ ARRANG -4.84             // La Palma, 01/12/2021
...
-*
+
 * [ Cherenkov emission parameters ]
-*
-CERSIZ  5.         // Not above 10 for super/ultra-bialkali QE; 7 is fairly OK; 5 should be safe.
+CERSIZ 5.         // Not above 10 for super/ultra-bialkali QE; 7 is fairly OK; 5 should be safe.
 CERFIL 0
-CWAVLG  230.  900.                     // Cherenkov wavelength band
-*
+CWAVLG 230.  900.                     // Cherenkov wavelength band
```
Comment (Voutsi) [inline config_LST1_muons.cfg:98]: Shall we update it to CWAVLG  270.  750. ?

**Comment by maxnoe** [comment]
@rlopezcoto should we merge this?

**Comment by rlopezcoto** [comment]
sure!

**Pull request merged**